kayenta-signalfx/src/main/java/com/netflix/kayenta/signalfx/canary/SignalFxCanaryScopeFactory.java

```diff
@@ -0,0 +1,61 @@
+/*
+ * Copyright (c) 2018 Nike, inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License")
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ */
+
+package com.netflix.kayenta.signalfx.canary;
+
+import com.netflix.kayenta.canary.CanaryScope;
+import com.netflix.kayenta.canary.CanaryScopeFactory;
+import org.springframework.stereotype.Component;
+
+import java.util.Map;
+import java.util.Optional;
+
+@Component
+public class SignalFxCanaryScopeFactory implements CanaryScopeFactory {
+
+  public static String SCOPE_KEY_KEY = "_scope_key";
+
+  @Override
+  public boolean handles(String serviceType) {
+    return "signalfx".equals(serviceType);
```

Comment:
Can use the constant from `SignalFxCanaryMetricSetQueryConfig`.

Reply:
- [ ] Use constant.